deployments/ansible/molecule/default/verify.yml

```diff
@@ -3,8 +3,13 @@
   hosts: all
   gather_facts: true
   tasks:
-    - name: Import shared verification tasks
-      ansible.builtin.import_tasks: ../shared/verify_tasks.yml
+
+    - name: Populate service facts
+      ansible.builtin.service_facts:
+
+    - name: Assert splunk-otel-collector service is stopped
+      assert:
+        that: ansible_facts.services['splunk-otel-collector.service'].state == 'stopped'
```

Comment:
Should this task be moved to `custom_vars` or a new test suite?  Otherwise, issues with starting the collector service may not get caught.

Reply:
this default test suite wasn't doing much with the collector. It wasn't starting it or using it, just checking config. There are other tests that run against the collector to make sure it can ingest logs. I figured I'd use this test.

Reply:
Ah, I guess we check the service status in the other tests, so hopefully any issues would get caught there.